velox/exec/OutputBuffer.h

```diff
@@ -114,6 +149,8 @@ class DestinationBuffer {
   // the callback.
   DataAvailable getAndClearNotify();
 
+  BufferInfo getInfo();
```

Comment:
Maybe BufferStats?

getInfo() -> stats()

Make this method const

Reply:
+1 for `BufferStats`.

Reply:
done.